app/models/auxiliar_business_unit.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AuxiliarBusinessUnit < ApplicationRecord
+  belongs_to :plan_item
+  belongs_to :business_unit
+
+  has_one :review, through: :plan_item
+  has_many :control_objective_items, through: :review
+
+  after_destroy :remove_all_scored_business_unit
+
+  private
+
+    def remove_all_scored_business_unit
+      control_objective_items.each do |co_i|
+        co_i.update(scored_business_unit_id: nil)
+      end
```

Comment:
```suggestion
      control_objective_items.update_all scored_business_unit_id: nil
```
Y es la última, lo prometo... jeje. Lo bueno de hacerlo así es que es una sola consulta, un UPDATE y listo.

Reply:
Franco no hay drama, olvidate. A mi me re sirve esto. Genial ahi lo cambie. Estos metodos auxiliares me falta mucho jajajaja